packages/docs/src/stories/components/tab.stories.ts

```diff
@@ -151,6 +151,38 @@ export const Disabled = {
   `
 };
 
+/**
+ * Use the `visually-disabled` attribute to style the component as if it was disabled and enable `aria-disabled` to allow it to be reachable by screen readers.
+ *
+ * __Hint:__ When using this attribute, make sure to provide ways to inform the user why the element is disabled and how to enable it. This can be done by wrapping the element in a sd-tooltip.
+ *
+ * **Accessibility Hint:** Disabling elements is not recommended for accessibility reasons.
+ */
+export const VisuallyDisabled = {
+  render: () => {
+    return html`
+      <sd-tab-group>
+        <sd-tab slot="nav" panel="tab-1" variant="default">Label</sd-tab>
+        <sd-tab-panel name="tab-1">
+          <div>
+            Tab panel 1 - Provident illo neque vel ex. Inventore perspiciatis delectus nisi doloremque soluta inventore.
+          </div>
+        </sd-tab-panel>
+        <sd-tooltip slot="nav" content="Visually Disabled" trigger="hover focus" size="sm" placement="top" hoist>
+          <sd-tab panel="tab-2" variant="default" visually-disabled>Disabled</sd-tab>
```

Comment:
@MartaPintoTeixeira should we also have `visually disabled` in the tab text? I think we already do this in other components